### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Monitoring_Keyboard: Focus is not going on any of the accessible components of the graph.

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#97]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS2.1.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Monitoring and press enter
6) Navigate to all the controls using Tab/Space Bar/Enter/Arrow keys or using Keyboard shortcut provided.
7) Activate the controls like button, link etc. using Enter/Space Bar Key.

**Actual Result:**
Focus is not going on any of the accessible components of the graph. The contents and controls present in the Monitoring page are inaccessible through keyboard

**Expected Result:**
User should be able to navigate to all the controls using keyboard.

**User Impact:** 
Visually Impaired Users or Keyboard users may face difficulty in navigating through the contents and controls if they are inaccessible through keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachments:**
[Bug_97_Page is not accessible through Keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323533/Bug_97_Page.is.not.accessible.through.Keyboard.pptx)

**Comment by jhkimnew** [comment]
This should be by design issue. We show only visual information which can't be converted to other information. If users strongly requests this information as other format, we should create other work item as a new feature.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.1.1; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
